assets/sass/components/_summary-card.scss

```diff
@@ -0,0 +1,20 @@
+.hmpps-summary-card {
+  border: 1px solid black;
```

Comment:
There doesn't seem to be a callout for border colour on the Figma design, but inspecting it I think it's #505A5F rather than "black"

Would be good to use the `govuk-colour` function to define these - #505A5F would be `dark-grey`

And finally, just my OCD... better to remove commented out lines if they are not needed any more.

Reply:
Good spot. Please replace black with  govuk-colour("dark-grey") or whatever colour is needed.
https://design-system.service.gov.uk/styles/colour/